tests/integration/components/feed-item-test.js

```diff
@@ -27,20 +29,40 @@ test('it renders', function(assert) {
   this.render(hbs`{{feed-item article=article }}`);
 
   assert.equal(
-    this.$('img').attr('src'),
-    'https://static.productionready.io/images/smiley-cyrus.jpg',
+    testSelector('article-author-image', article.author.image),
+    '[data-test-article-author-image="https://static.productionready.io/images/smiley-cyrus.jpg"]',
     'Image is default image'
   );
 
-  assert.equal(this.$('a.author').text().trim(), 'Alon Bukai', 'Author name is correct');
+  assert.equal(
+    testSelector('article-author-username', article.author.username),
```

Comment:
I'm not sure this is testing what we actually want to test here. The idea isn't to verify the data based on the selectors, but to use the selectors to locate the DOM elements.  We'd want assert the text of the element or something like that, not that the attribute has the right value.